README.md

```diff
@@ -160,6 +160,8 @@ The script has the responsibility to:
 - Add those changes to git
 - Commit those changes to git
 
+Example scripts can be found in [next-transformations](https://github.com/Financial-Times/next-transformations).
```

Comment:
it's worth noting that nori is a public repo, as we'd thought it could be useful outside the FT, but `next-transformations` is private. we could make that repo public,  i don't think there's anything sensitive in it, or maybe copy an example or two into an `examples` folder in this repo?

Reply:
I have pulled in the script I recently added to `next-transformations` as it feels pretty minimal and instructive.